action-templates/actions/action-checkout/action.yml

```
@@ -1,8 +1,27 @@
+  python:
+    description: "name of the artifact upload"
+    value: ${{ steps.filter.outputs.python }}
+    
 runs:
   using: "composite"
   steps:
     - name: Checkout code
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+    - name: Check changed files and directories
+      id: filter
+      uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)